src/components/image/CdrImg.vue

```diff
@@ -85,6 +85,10 @@ const props = defineProps({
   cover: {
     type: Boolean,
   },
+    /**
+   * Adds a custom class to the cdr-img__ratio container div
+   */
+  containerClass: String,
```

Comment:
would it make it more clear to call this ratioContainerClass or this added for all modifiers?

Reply:
My thinking was that consumers are using this class in ways unrelated to the ratio styles, so it felt a little more confusing to me to scope it as such